core/controllers/platform_feature_test.py

```diff
@@ -217,22 +214,15 @@ def tearDown(self) -> None:
 
         super().tearDown()
 
-    def _set_dummy_feature_status_for_mode(
-        self, is_enabled: bool, mode: param_domain.ServerMode
-    ) -> None:
+    def _set_dummy_feature_status(
+        self, is_enabled: bool) -> None:
```

Comment:
feature_is_enabled (since it's a bool and not a function, don't start with a verb)

Reply:
Done